docker/Dockerfile.nosgx

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:18.04
+
+RUN echo "APT::Acquire::Retries \"5\";" | tee /etc/apt/apt.conf.d/80-retries
+
+COPY getting_started/setup_vm/ /setup_vm/
+RUN apt update
```

Comment:
Better to combine this `RUN` and the one below to minimise the number of layers?

Reply:
Sure